packages/calcite-components/src/components/slider/slider.tsx

```diff
@@ -263,7 +263,7 @@ export class Slider
   @property() minValue: number;
 
   /**
-   * When `true`, the slider will display values from high to low.
+   * When present, the slider will display values from high to low.
```

Comment:
Another potential follow-up PR?: 

Should we change "slider" out to "component" for consistency?